packages/forms/src/patterns/input/builder.ts

```diff
@@ -0,0 +1,12 @@
+import { PatternBuilder } from '../../pattern';
+import { type InputPattern } from '.';
+
+export class Input extends PatternBuilder<InputPattern> {
```

Comment:
I love the concept of the builder class here. Could we document the reasoning in an ADR?

Reply:
Will do. As it's more technical, I think it might read better as README text.